src/ccip/tests/api/test_ccip.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+endpoint = "/ccip/"
+
+
+def assert_data_structure(data, key):
+    assert key in data
+    items = data.get(key, [])
+    for item in items:
+        assert 'id' in item
+        assert 'zh' in item
+        assert 'en' in item
+        assert 'name' in item['zh']
+        assert 'name' in item['en']
+        if key == "speakers":
+            assert 'avatar' in item
+            assert 'bio' in item['zh']
+            assert 'bio' in item['en']
+
+
+@pytest.mark.django_db
+def test_data_structure(client):
+    response = client.get(endpoint, follow=True)
+    assert response.status_code == 200
+    data = response.json()
+
+    assert_data_structure(data, 'session_types')
+    assert_data_structure(data, 'tags')
+    assert_data_structure(data, 'rooms')
+    assert_data_structure(data, 'speakers')
+
+
+@pytest.mark.django_db
+def test_session(client):
```

Comment:
**suggestion:** this test seems to test the data schema too, which has the same purpose as above test. Do you consider putting them in the same test function and validating the schema according to the different entities? Probably we could also clarify the purpose of this test by naming it as `test_session_schema`

Reply:
OK